src/pages/home/report/ReportActionItem.js

```diff
@@ -287,9 +285,4 @@ class ReportActionItem extends Component {
 
 ReportActionItem.propTypes = propTypes;
 ReportActionItem.defaultProps = defaultProps;
-
-export default withOnyx({
-    draftMessage: {
-        key: ({reportID, action}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${action.reportActionID}`,
```

Comment:
@yuwenmemon I'm not sure if this would improve things much but we could try subscribing to all the drafts just once in `ReportActionsView` under the key

```js
draftMessages: {
    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS,
},
```

then pass the `draftMessage` as a prop like 

```jsx
const key = `${${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${reportActionID}}`;
const draftMessage = this.props.draftMessages[key];

... 

<ReportActionItem
    draftMessage={draftMessage}
    ...
/>
```

Not sure if this would have any bad effects while in edit mode, but might be something to try.

Reply:
Just tested on my physical device and confirmed that on this branch with the above changes it takes quite a while after tapping `edit comment` for the UI to let you edit the comment, whereas on main it lets you edit immediately